### DISCUSSION TITLE
Changing default editor

### DISCUSSION THREAD

**Discussion by Geegodanne** [community/community#58065] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am setting up GitHUb. It opens up with defalut editor as Notepad ++, I want to us Sublime Editor. How do I make that change?

**Comment by DSDmark** [comment]
Hi @Geegodanne,

It is not related to GitHub issue. It's your system default setting. You could try to change your default setting on your system according to your **OS**.

**Reply by Geegodanne** [reply]
I changed my editor to Sublime Text and set it to always open the file type. I am using chrome browers on windows 11. Nothing changed in Github. There is no option to make the change in Github. The dropdown only shows Notepad++.

**Reply by Geegodanne** [reply]
I also have no idea how Github selected Notepad++ as the editor.
